application/views/scripts/about/index.phtml

```diff
@@ -104,10 +104,36 @@ use ipl\Web\Widget\StateBadge;
             } catch (Throwable $e) {
                 // suppress
             }
-        if ($hasPending): ?>
-            <div class="pending-migrations clearfix">
-                <h2><?= $this->translate('Pending Migrations') ?></h2>
-                <table class="name-value-table migrations">
+        ?>
+            </div>
+        </div>
+        <?php $config = Config::app()->getSection('global') ?>
+        <div class="pending-migrations clearfix">
+            <?php if($mm->getPendingMigrations() || (!isset($config->config_resource))) : ?>
+            <h2><?= $this->translate('Pending Migrations') ?></h2>
+            <?php endif ?>
+            <table >
+                <tr>
+                    <th>
+                  <?php  if (!isset($config->config_resource)) : ?>
+                      <?=  new EmptyState(
+                              $this->translate('No Configuration Database selected. To establish a valid database connection set: ')
+                    )
+                  ?><td>
+                  <?= $this->qlink(
+                      $this->translate('Configuration Database'),
+                      'config/general/',
+                      [],
+                      ['title' => sprintf($this->translate('Go to Application/General')), 'Configuration Database']
+                  ) ?>
+                    </td>
```

Comment:
```suggestion
                      <?=  new EmptyState(
                              HtmlString::create(sprintf(
                                      $this->translate('No configuration database is selected. To establish a valid database connection set the %s'),
                                      new Link($this->translate('Configuration Database'), 'config/general')
                              ))
                    )
                  ?>
```

`HtmlString` can render the html in the string. And it looks better to me.
You need to add `ipl\Html\HtmlString` and `ipl\Web\Widget\Link` at the top of the file.

Reply:
You need to check why this is happening. The issue appears to be caused by the current CSS rules, which are preventing the links from displaying on the same line. Consider updating the layout properties that are causing this.